lib/utils/couchclient/src/index.js

```diff
@@ -400,6 +400,10 @@ const couchclient = (partitionFn, uriFn, connectionFn, errorDBFn) => {
           });
     },
 
+    distinctValues: () => {
+      throw Error('Not Implemented');
```

Comment:
The idea is that this way it will be more disruptive and the execution will be aborted. The call back error is easier to be neglected.